src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/MethodTypeInference.cs

```diff
@@ -1723,6 +1723,11 @@ private bool LowerBoundConstructedInference(TypeSymbol source, TypeSymbol target
                 return false;
             }
 
+            if (constructedTarget.IsTupleType)
+            {
+                constructedTarget = ((TupleTypeSymbol)constructedTarget).UnderlyingTupleType;
```

Comment:
Are we losing tuple element names here in type inference? If so, please add a PROTOTYPE comment.